src/app/ui/NewTaskForm.tsx

```diff
@@ -29,28 +33,90 @@ export const NewTaskForm = ({ handleCreate }: { handleCreate: () => void }) => {
     item: assignee[0],
     type: SelectorType.ASSIGNEE_SELECTOR,
   })
+  const { renderingItem: _templateValue, updateRenderingItem: updateTemplateValue } = useHandleSelectorComponent({
+    item: undefined, //initially we don't want any value to be selected
```

Comment:
`item:undefined` and not providing the item key at all is same but since `item` is not optional, we can't just not provide the item key at all in this case. 
I am using `undefined` because according to the TypeScript's guideline it is suggested to use `undefined` for better type safety. [see here](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined). Can you let me know why using `null` provides better type safety here in this case?